src/platform/Darwin/ThreadStackManagerImpl.cpp

```diff
@@ -0,0 +1,933 @@
+/*
```

Comment:
Why is this code here?  It's not compiled, and can't possibly work since it's tied to various glib/Linux stuff.

The right thing to do for the YAML is probably to have it conditioned on PICS and to not claim support for this at all on Darwin if it's not supported there.

Reply:
this is an accident